src/commands/wait_till_ready_postgres.yml

```diff
@@ -0,0 +1,19 @@
+---
+description: "Wait for postgres to be ready. Defaults to port 5432 on localhost."
+parameters:
+  port:
+    type: integer
+    default: 5432
+  host:
+    type: string
+    default: "localhost"
+steps:
+  - run:
+      name: Wait for port to be ready
```

Comment:
I'd maybe rename this step `Wait for postgres to be ready`?

Reply:
agreed, it was a copy and paste from the wait till ready executor :grin: